### DISCUSSION TITLE
Different text sizes makes them vertically unaligned

### DISCUSSION THREAD

**Discussion by Almeida-Oco** [hecrj/iced#864] [Q&A]
Hello! I found this project rather interesting and decided to try it and play around with it. 
I noticed something weird when having text on the same "line" but with different sizes. I tried it on version `0.3.0` with the code:

```
use iced::{Color, Element, Length, Row, Sandbox, Text, VerticalAlignment};

fn main() -> iced::Result {
    Hello::run(iced::Settings::default())
}

struct Hello;

impl Sandbox for Hello {
    type Message = ();

    fn new() -> Self {  Hello  }
    fn title(&self) -> String {  "Hello World".to_string()  }
    fn update(&mut self, _message: Self::Message) { }

    fn view(&mut self) -> Element<'_, Self::Message> {
        let content: Element<()> = Row::new()
            .height(Length::Units(60))    // Different height values do not seem to affect it
            .push(
                Text::new("Hello")
                    .size(20)
                    .height(Length::Fill)
                    .vertical_alignment(VerticalAlignment::Bottom),
            )
            .push(
                Text::new("World")
                    .size(50)
                    .height(Length::Fill)
                    .vertical_alignment(VerticalAlignment::Bottom),
            )
            .into();

        content.explain(Color::BLACK)
    }
}
```

Whether I vertically align the text to the bottom or to the top, the base line of the text is different. In the code above, you can see the "Hello" part is lower than the "World". I was expecting that both texts would have the same bottom/top alignment even with different sizes but that was not the case.

Is there some other way to have this differently sized text in the same line that I missed or is this unintended behavior?

**Comment by 13r0ck** [comment]
[This might answer your question](https://youtu.be/Ie-gqwSHQr0?t=724), its a talk from a different library in a different language, but the point still stands. The video later talks about his solution, but the problem he presents exists in both CSS and Iced. Text can be very weird. Unfortunately you are just going to have to manually align the text.
